### ISSUE TITLE
Distribute on NPM/registry?

### ISSUE THREAD

**Issue by Niek** [tanishqmanuja/apkmirror-downloader#11]
It would be great if you could distribute releases through NPM or a different registry, so people can simply run `npx -y apkmirror-downloader`.

**Comment by Niek** [comment]
FYI referencing the GH repo give this error:
```bash
npx -y github:tanishqmanuja/apkmirror-downloader
sh: apkmd: command not found
```

**Comment by tanishqmanuja** [comment]
Not planned because it's technically is a violation of apkmirror's terms of use. I guess that's the main reason many big open source projects do web scrap on apkmirror but never release the tool wide open in an easily accessible manner.

**Comment by Yakov5776** [comment]
Change of thought?

**Comment by tanishqmanuja** [comment]
Yes, Thanks to you for showing me how different projects already using scrapping and apkmirror didn't bother to take any action.

**Comment by Niek** [comment]
Nice! It's not working with `npx` though - would it be possible to fix that?

**Comment by tanishqmanuja** [comment]
Weird issue on npm side, I have no idea about npm but both pnpm dlx and bunx work as usual.

If you know any suggested fixes for npm, I can definitely try to fix. Otherwise I personally don't use npm anyways.
